### DISCUSSION TITLE
How to draw rectangle canvas on camera with .Net MAUI?

### DISCUSSION THREAD

**Discussion by omarcrav** [dotnet/maui#12301] [General]
I try draw a rectangle on camera View to center a card ID and return image cropped. I done previously with Xamarin Forms with a Redenrers to IOS and Android, but libraries already deprecated or obsolete.

I want help to another method or package to resolve it. I search some days, but I haven't found a solution.

P.D. I don't need recognize card ID info, only need crop image.

**Comment by ubheamar** [comment]
Hello Omar,

I am also searching same thing. let me know if you found something

**Reply by alfredotsco** [reply]
Hi @ubheamar, @omarcrav 

Any updates?
